var/spack/repos/builtin/packages/miopen-hip/package.py

```diff
@@ -154,6 +154,7 @@ class MiopenHip(CMakePackage):
         "5.2.0",
         "5.2.1",
         "5.2.3",
+        "5.3.0",
```

Comment:
Does this package (`miopen-hip`) also need to have `5.4.0`?

Reply:
yes. i am currently working on it. But on the mainline there was missing 5.3.0 label missing for this recipe which i have now corrected. I am observing miopen-hip has more changes for 5.4.0 and is in progress now.